_includes/landing-page/popular-videos.html

```diff
@@ -6,19 +6,22 @@ <h5 class="title">Popular videos</h5>
         <p>Watch the most popular videos from DockerCon.</p>
       </div>
     </div>
+    <!-- How to Get Started with Docker -->
     <div class="row">
       <div class="col-xs-12 col-md-6 col-lg-12">
         <a class="video-item" href="https://youtu.be/iqqDU2crIEQ?t=30" target="_blank" rel="noopener">
           <img class="video-image" src="/images/video.svg" alt="video: How to get started with Docker" width="65" height="44" />
           <div class="video-title">How to get started with Docker</div>
```

Comment:
Hmm.. not sure if this adds much value; the same text is already a few lines below the comment, and adding these comments will actually introduce more work to update them together with the actual HTML.

<img width="1248" alt="Screenshot 2021-11-11 at 15 20 28" src="https://user-images.githubusercontent.com/1804568/141314008-48b17d3f-d700-4c20-a4a9-d54f6576f523.png">